frontend/src/app/oe/interfaces/oe-energy.interface.ts

```diff
@@ -155,6 +155,20 @@ export interface BlockTimeStrikeGuessPublic {
   guess : 'slow' | 'fast';
 }
 
+export interface BlockTimeStrikeGuessResultPublic {
```

Comment:
there is no API calls that return such data anymore. Please refer to `$strikesGuessesWithFilter` API call for reference of it's return type.